### PR TITLE
PHOENIX-5252 Add job priority option to UpdateStatisticsTool

### PR DESCRIPTION
**Pull request by yanxinyi**

### CODE REVIEW

**Comment by karanmehta93** [comment]
LGTM overall, @yanxinyi Can you provide a brief description on the Jira as to how this option works internally so that the users understand exactly how to use it?

File: phoenix-core/src/main/java/org/apache/phoenix/schema/stats/UpdateStatisticsTool.java [hunk 1]
```diff
@@ -164,12 +168,36 @@ void parseArgs(String[] args) {
...
+             return JobPriority.DEFAULT;
+         }
+
+         switch (jobPriorityOption) {
+             case "0" : return JobPriority.VERY_HIGH;
+             case "1" : return JobPriority.HIGH;
+             case "2" : return JobPriority.NORMAL;
+             case "3" : return JobPriority.LOW;
+             case "4" : return JobPriority.VERY_LOW;
+             case "5" : return JobPriority.DEFAULT;
```
Comment (ChinmaySKulkarni) [inline phoenix-core/src/main/java/org/apache/phoenix/schema/stats/UpdateStatisticsTool.java]: What is the `JobPriority.DEFAULT` level? Perhaps this should be the default case?

Comment (yanxinyi) [reply phoenix-core/src/main/java/org/apache/phoenix/schema/stats/UpdateStatisticsTool.java]: https://issues.apache.org/jira/browse/MAPREDUCE-5870

Yes, if priority is not indicated by the user, it sets to default.

Comment (yanxinyi) [reply phoenix-core/src/main/java/org/apache/phoenix/schema/stats/UpdateStatisticsTool.java]: maybe I should change to normal or default instead UNDEFINED_PRIORITY

**Pull request closed without merging**